chart/Chart.yaml

```
@@ -19,7 +19,7 @@
 ---
 apiVersion: v2
 name: airflow
-version: 1.7.0-dev
+version: 1.8.0-dev
```

```suggestion
version: 1.7.0-dev
```

This should still be 1.7.0-dev. This gets bumped as part of the release process.